libs/teams/teams-chat-workflow-spring-boot-starter/src/main/java/org/finos/springbot/teams/bot/BotController.java

```diff
@@ -0,0 +1,108 @@
+package org.finos.springbot.teams.bot;
+
+
+import com.microsoft.bot.builder.Bot;
+import com.microsoft.bot.connector.authentication.AuthenticationException;
+import com.microsoft.bot.integration.BotFrameworkHttpAdapter;
+import com.microsoft.bot.schema.Activity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+/**
```

Comment:
Can you attach some javadoc to say where these classes come from and why we've added them rather than used the dependency?

thanks